srcpkgs/python3-pathvalidate/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-pathvalidate'
+pkgname=python3-pathvalidate
+version=2.5.0
+revision=1
+short_desc="Python library to sanitize/validate strings such as filenames/paths"
+maintainer="Nikolay Belikov <nb@nbelikov.com>"
+license="MIT"
+homepage="https://github.com/thombashi/pathvalidate"
+changelog="https://github.com/thombashi/pathvalidate/releases"
+
+build_style=python3-module
+wrksrc="pathvalidate-${version}"
+hostmakedepends="python3-setuptools"
+depends="python3>=3.6"
```

Comment:
```suggestion
depends="python3"
```
it is extremely unlikely that `python3` would get downgraded below 3.6 at any time.